pages/learn/manage/actions.md

```diff
@@ -32,14 +32,21 @@ You may also enable or disable public device URLs by clicking the _Public device
 
 The `Restart Application` action restarts the currently running **application containers**. Your application's running containers will be removed and recreated from scratch. This behavior is intended, and is different from running `balena restart [OPTIONS] CONTAINER [CONTAINER...]` in a host OS terminal instance from your dashboard, which will not remove your containers. If you are trying to persist data between container removals, see [persistent storage][persistent-storage] for strategies.
 
-When the containers are stopped, the application is politely asked to stop by sending a `SIGTERM` and after 10 seconds of wait time a `SIGKILL` is sent.
+When the containers are stopped, the application is politely asked to stop by sending a `SIGTERM`. After 10 seconds of wait time, a `SIGKILL` is sent.
```

Comment:
```suggestion
When the containers are stopped, the application is politely asked to stop by sending a `SIGTERM`. If the application hasn't stopped after 10 seconds a `SIGKILL` is sent.
```

Reply:
Does it actually try to detect if the application has stopped, or does it just send the SIGKILL regardless?

Reply:
Good point & nice catch!. I rephrased this bit without thinking about the functionality underneath.

Reply:
@lurch SIGKILL would only be sent if the container(s) is/are still running, since the command is much like Docker's `docker stop` command.